src/components/organisms/UiSidePanel/UiSidePanel.stories.scss

```diff
@@ -40,3 +21,40 @@
 .condition-section {
   --icon-color: var(--color-text-action-primary);
 }
+
+//<Code id="side-panel-condition-details">
+.side-panel-with-condition-details {
```

Comment:
This is the way to create styles for advanced stories. Add a new class to create new scope for styles.